### ISSUE TITLE
[Dark Mode] HUD background color

### ISSUE THREAD

**Issue by danielebogo** [wordpress-mobile/WordPress-iOS#12595]
Modal HUD shown after hitting Continue on New Account screen should use system grey background.

![IMG_1143](https://user-images.githubusercontent.com/1396552/64465852-c059bd80-d0dc-11e9-964e-66827188fd7a.PNG)

**Comment by elibud** [comment]
@frosty is this one still an issue? Can someone in your team please look into it?

**Comment by frosty** [comment]
Yes, looks like this is a global setting for everywhere we use SVProgressHUD. Should be a fairly quick fix.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by jkmassel** [comment]
This issue is still reproducible. An easy way to trigger it: open any media item from the Media section, then delete it. The HUD that appears ("Deleting…" / "Deleted!") uses the hardcoded dark background regardless of system appearance.
